package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "husky": "^8.0.1",
```

Comment:
You should add husky and those testing libraries as a Dev Dependencies so they won't be added to your production builds